salt/modules/yumpkg.py

```diff
@@ -1056,9 +1056,10 @@ def install(name=None,
     return ret
 
 
-def upgrade(refresh=True, fromrepo=None, skip_verify=False, **kwargs):
+def upgrade(name=None, refresh=True, fromrepo=None, skip_verify=False, pkgs=None, normalize=True, **kwargs):
```

Comment:
That totally makes sense to me.  The API change from something counterintuitive to something safer may be worth more than the cost of it being backwards incompatible.  What do you think @cachedout?